bundle/downstream/manifests/quay-bridge-operator-quay-bridge-operator-netpol_networking.k8s.io_v1_networkpolicy.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: quay-bridge-operator-quay-bridge-operator-netpol
+  namespace: quay-bridge-operator-system
```

Comment:
is this always going to be the same? Can't they install into different namespaces?